dcos/emitting.py

```diff
@@ -74,7 +74,7 @@ def print_handler(event):
         _page(event, pager_command)
 
     elif isinstance(event, errors.Error):
-        print(event.error(), file=sys.stderr)
+        print(event.error(), file=sys.stderr, flush=True)
```

Comment:
Let's see what the tests say but I don't think this works with python 2.7: https://docs.python.org/2.7/library/functions.html#print